app/Scrambler.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable-all
+
+from random import choice, shuffle
+
+from Picture import Picture
+from Tile import Tile
+
+
+class Scrambler:
```

Comment:
The `Scrambler` class would also benefit from a docstring that explains its purpose.

Reply:
Docstring is now added